### ISSUE TITLE
Prevent automatic switching of desktops when opening applications

### ISSUE THREAD

**Issue by dsp0105** [mzomparelli/zVirtualDesktop#172]
Issue:
------
Automatic switching of desktops when opening applications. This is really annoying and I have to double check the desktop number every time I open an application. 

Can we prevent automatic switching of desktop when opening an application? Thanks!

**Comment by mzomparelli** [comment]
I've been trying to come up with a solution to this.

Here's what I'm thinking:

-I'm able to tell if it was you that switched or if it switched automatically
-I can move you back to the desktop that you were on while bringing the window with

The problem with this is many developers are not programming with virtual desktops in mind. Programs like Chrome will open links in an instance that is already open. In reality, Chrome should check if there is an instance on the desktop and if not it should open a new one even if there is an instance on another desktop. When this happens I would move the entire Chrome window and all of its tabs. This might be undesired functionality for some users.

**Comment by mzomparelli** [comment]
https://github.com/mzomparelli/zVirtualDesktop/issues/49

**Comment by dsp0105** [comment]
Yeah, I see the fundamental issue with chrome/applications not doing the check you mentioned. 

Your workaround sounds good, the steps I see are like this:

-  User clicks on an applicaton

- Let Windows OS switch to an other desktop and open the application

- Then you switch back to the original desktop along with the application through zVD, then the user can 
   decide what he wants to do with the application : 

    a) He can create a new instance of application manually and move the "moved application" to its 
        original desktop,    
    
    OR

    b) Continue using the current instance of the opened application which was moved from other 
        desktop by zVD

Thanks for your fast feedback!

**Comment by mzomparelli** [comment]
The other annoying part is that when the OS switches itself it does so with the animation. I could force the animation back or not do it at all when switching back. Doing the animation twice could be dizzying. If that's even a word. Not showing the animation going back could confuse the user.

**Comment by dsp0105** [comment]
I see, agree that would be disconcerting to the user with all the screen flashing. 
I disable animations in Windows performance options (system->performance->optimize for best performance). Hopefully that would help avoid the screen flashing.
